src/app/parcels/fetchParcelTableData.ts

```diff
@@ -273,9 +273,16 @@ export const getParcelsByIds = async (
         throw new DatabaseError("fetch", "parcel table", logId);
     }
 
-    const congestionCharge = await getCongestionChargeDetailsForParcels(data, supabase);
+    const { data: congestionChargeDetails, error: congestionChargeError } =
+        await getCongestionChargeDetailsForParcelsTable(data);
+
+    if (congestionChargeError) {
+        const logId = await logErrorReturnLogId("Errr");
+        throw new EdgeFunctionError("congestion charge check", logId);
+    }
+
     const { parcelTableRows, error: processParcelDataError } =
-        await processingDataToParcelsTableData(data, congestionCharge);
+        await processingDataToParcelsTableData(data, congestionChargeDetails);
 
     if (processParcelDataError) {
         throw new Error("Failed to process parcels.", { cause: processParcelDataError });
```

Comment:
I did do some reordering of functions in this file which I think is what you can see above - mainly because I got some errors saying things were called before they were defined (not totally sure why?) so I tried to put them in order

Reply:
Otherwise this is me changing the data structure that is being passed through to contain { data, error } so the error could be rendered

Reply:
Seems fine now, I was originally trying to do it so it would still show the data and also the error message if it was just congestion charge missing, so maybe it came up at that stage